packages/client/src/components/TrecipePicker/trecipePicker.scss

```diff
@@ -0,0 +1,78 @@
+@import '../../components/variables';
+@import '../../components/mixins';
+
+.trecipe-picker {
+    width: 21.5rem;
+    height: 21.5rem;
+    text-align: left;
+    margin: 15px 25px;
+    font-family: $default-font-family;
+    overflow: hidden;
+    ::-webkit-scrollbar {
+        width: 0;
+    }
```

Comment:
This does not work on Firefox 😅 A while ago, I tried to customize the scroll bar on map page but I gave up after reading [this](https://css-tricks.com/the-current-state-of-styling-scrollbars/) and realized it is hard to make the scrollbar consistent across different browsers haha 

![image](https://user-images.githubusercontent.com/36020702/89092620-217b2700-d368-11ea-833d-342882c87aaf.png)

Reply:
Ooof. Why don't people just stick to Chrome LOL
I meannnn if we really want cross browser, then we can look into something like [SimpleBar](https://github.com/Grsmto/simplebar). 
I think I'll keep the scrollbar hiding for chrome for now, imo looks better for the demo hahaha.

Reply:
I wish all browsers display webpage the same way. Every time I switched from Firefox to Chrome I get shocked at how different Trecipe look, and everything looks bigger in Chrome 😂
LOL let's not go down that route, it is just a scrollbarr